hoot-core-test/src/test/cpp/hoot/core/ops/FindIntersectionsOpTest.cpp

```diff
@@ -62,6 +62,11 @@ class FindIntersectionsOpTest : public CppUnit::TestFixture
 
 public:
 
+  void setUp()
```

Comment:
All it does is make the VM able to run `HootTest --quick` right after a call from `vagrant up`.  It is a bug I found while working on the other stuff.